client/src/components/atoms/InlineComment.vue

```diff
@@ -175,7 +175,7 @@ defineExpose({
 
 <style lang="sass" scoped>
 .q-card.active
-  box-shadow: inset 0 0 5px 2px yellow, 0 1px 5px rgba(0, 0, 0, 0.2), 0 2px 2px rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12)
+  box-shadow: inset 0 0 5px 2px #F8DB8B, 0 1px 5px rgba(0, 0, 0, 0.2), 0 2px 2px rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12)
```

Comment:
Something to think about is should we start using rem instead of px in our css?

Reply:
Quasar has a mixed use of rems and pixels for its built-in styles in its source code. It also specifies the base font size as `14px` which is not useful for pixel-to-rem conversions. This should've been set to `62.5%` instead. Trying to figure out how to implement a workaround and adapt the framework for manageable numbers in rems is not a problem that excites me. 

Unless we can figure out a solution to Quasar's mistake and also identify specific problems caused by absolute length units when users adjust their _text_ size (which is different and more difficult to do than adjusting _zoom_), I say we just continue to use pixels and rems as we please.